### DISCUSSION TITLE
Am I blocked from the blog comments?

### DISCUSSION THREAD

**Discussion by Secret-chest** [linuxmint/discussions#689] [Questions]
Lately, every time I've tried to post a comment to blog.linuxmint.com, nothing happened.

**Comment by Palladini** [comment]
No idea why you are saying that.  I replied

**Comment by mtwebster** [comment]
You're in a queue - don't worry I end up there as well, every time. I'm pretty sure it's a result of 'securi' not trusting certain domains as much as others.
